util/redis_helper.py

```diff
@@ -4,3 +4,6 @@
 def init_redis():
   global redis_client
   redis_client = Redis.from_url(os.environ['REDIS_URL'])
```

Comment:
nit: maybe add a TODO for setting max_connections in the  constructor of the Redis client.